src/components/views/HeightView.js

```diff
@@ -2,42 +2,29 @@ import React, {Component} from "react";
 import {Header} from "semantic-ui-react";
 import {Helmet} from "react-helmet";
 import {Line} from 'react-chartjs-2';
+import * as gpsUtil from "../../util/gps.js";
 
 class HeightView extends Component {
-
-    distance(elementOne, elementTwo) {
-        // Source: http://www.guymon.de/wordpress/2010/01/25/entfernung-zwischen-zwei-latlong-werten-berechnen/
-        var lat_1 = elementOne.latitude;
-        var lat_2 = elementTwo.latitude;
-        var lon_1 = elementOne.longitude;
-        var lon_2 = elementTwo.longitude;
-        var rho = 3960.0;
-        var phi_1 = (90.0 - lat_1)*Math.PI/180.0;
-        var phi_2 = (90.0 - lat_2)*Math.PI/180.0;
-        var theta_1 = lon_1*Math.PI/180.0;
-        var theta_2 = lon_2*Math.PI/180.0;
-        var d = rho*Math.acos( Math.sin(phi_1)*Math.sin(phi_2)*Math.cos(theta_1 - theta_2) + Math.cos(phi_1)*Math.cos(phi_2) );
-        return 1.609344*d;
-    }
-
     render() {
         const {gpsData} = this.props;
         const array = [];
         const labels = [];
+
         let currentDistance = 0.0;
         let lastElement  = null;
         gpsData.map(element => {
             let distance = 0.0;
             if (lastElement === null) {
                 distance = 0.0;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Review this useless assignment: "distance" already holds the assigned value along all execution paths. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS4165)